.github/workflows/build.yml

```diff
@@ -8,14 +8,13 @@ jobs:
     permissions:
       contents: read
     steps:
-      - name: Set up Go 1.x
-        uses: actions/setup-go@v2
-        with:
-          go-version: '1.15.8'
       - name: checkout
         uses: actions/checkout@v2
-      - name: lint
-        run: make lint
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v2
+        with:
+          version: v1.28.3
+          args: --timeout 10m
```

Comment:
Why timeout for a linter is so big? Can we decrease it to 3-5 minutes?

Reply:
Sometimes it takes a long time, I'd rather have a long timeout to begin with.